src/commands/init.js

```diff
@@ -150,7 +135,8 @@ git remote add origin https://github.com/YourUserName/RepoName.git
       this.error(repo.error)
     }
 
-    if (!siteData) {
+    // TODO: remove siteInfo.id === undefined once https://github.com/netlify/build/pull/2081 is published
```

Comment:
`@netlify/config@2.4.1` was published with that change.

Reply:
Done in https://github.com/netlify/cli/pull/1607/commits/f13d566f663f8bcfcf6ae7727544a96bd7051733